ccd-definition/CaseEventToFields/CareSupervision/removeOrder.json

```diff
@@ -50,7 +63,7 @@
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y",
     "FieldShowCondition": "orderTitleToBeRemoved=\"DO_NOT_SHOW\"",
-    "PageShowCondition": "removableType=\"ORDER\" OR removableType=\"ADDITIONAL_APPLICATION\""
+    "PageShowCondition": "removableType=\"ORDER\" OR removableType=\"ADDITIONAL_APPLICATION\" OR removableType=\"SENT_DOCUMENT\""
```

Comment:
Think this will cause the removeStandardDirectionOrderWarning to be shown which I think says "the case state will return to gatekeeping" - just want to double check this is intended?

Reply:
It is a PageShowCondition which controls the visibility of the second page. I think it is not related to the field `showRemoveSDOWarningFlag`.